docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml

```diff
@@ -128,5 +128,101 @@ required:
   - certifier
 example: |-
   {
-    "issue": "https://github.com/w3c-ccg/traceability-vocab/issues/238"
+    "id": "http://example.org/credentials/",
+    "type": [
+      "VerifiableCredential"
+    ],
+    "issuanceDate": "2021-02-04T20:29:37+00:00",
+    "issuer": "did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
+    "credentialSubject": {
+      "type": "USMCACertificateOfOrigin",
+      "certifier": "Importer",
+      "exporterDetails": {
+        "type": "Organization",
+        "name": "Torphy, Fahey and Stamm",
+        "description": "Synergized composite data-warehouse",
```

Comment:
ideally these would be more realistic.

Reply:
I don't disagree. I don't know if I'll have time to generate nice test data by today, though, so might fix this later. 
It's the kind of generator-data we used to have all over the place, so not worse that what we're used to. :)